rules/TypeDeclaration/Rector/ClassMethod/StringReturnTypeFromStrictScalarReturnsRector.php

```diff
@@ -25,40 +25,39 @@
 final class StringReturnTypeFromStrictScalarReturnsRector extends AbstractScopeAwareRector implements MinPhpVersionInterface
 {
     public function __construct(
-        private readonly StrictScalarReturnTypeAnalyzer $strictScalarReturnTypeAnalyzer,
         private readonly ClassMethodReturnTypeOverrideGuard $classMethodReturnTypeOverrideGuard,
-        private readonly StaticTypeMapper $staticTypeMapper
+        private readonly BetterNodeFinder $betterNodeFinder,
     ) {
     }
 
     public function getRuleDefinition(): RuleDefinition
     {
-        return new RuleDefinition('Change return type based on strict string scalar returns', [
+        return new RuleDefinition('Add string return type based on returned string scalar values', [
             new CodeSample(
                 <<<'CODE_SAMPLE'
 final class SomeClass
 {
-    public function foo($condition, $value)
+    public function foo($condition)
     {
-        if ($value) {
+        if ($condition) {
             return 'yes';
         }
 
-        return strtoupper($value);
+        return 'no';
     }
 }
 CODE_SAMPLE
                 ,
                 <<<'CODE_SAMPLE'
 final class SomeClass
 {
-    public function foo($condition, $value): string;
+    public function foo($condition): string;
```

Comment:
Typo: `;` after return type 

```suggestion
    public function foo($condition): string
```

Reply:
Thanks, will fix :+1: